assets/script/index.js

```diff
@@ -0,0 +1,122 @@
+// Assigments
+// #1
+// Write a function called isEven which checks if a number is even or odd.
+// Hints: 0 if even and 1 is odd, for any number N, its evennes is the same as N-2
+// Notice how this works when N is negative, try to fix it
+
+const n = prompt('Enter any number')
+function isEven () {
+  if (n % 2 === 0) {
+    return 'Odd'
+  } else {
+    return 'Even'
+  }
+} console.log(isEven())
+
+// #2
+// Write a function called countBs that takes a string as its only arguments and returns
+// a number that indicates how many uppercase "B" characters are in the string
+// countBs("Bombastic") => 1
+// countBs("bbbbBBBBonB") => 5
+
+// const str = prompt('Write => Haaland is wHell')
+// let count = 0
+// let position = str.indexOf('H')
+// while (position !== -1) {
+//   count ++
+//   position = str.indexOf('H', position + 1)
+// }
+// console.log(count)
+
+// #3
+// Write a function called CountChar that behaves like CountBs expect it takes a second
+// arguments that indicates the character that is to be counted
+// conutChar("Car", "C") =>1
+// countChar("capricious", "c") =>2
+
+// let countChar = 'Haaland holds the ball with holly legs';
+// function countChar(arg, Haaland)
+// let posi = string.indexOf("l")
+
+// #4
+// Write the function to generate the first n fabonacci numbers
+// ex 1 1 2 3 5
+// let n = 5;
+
+// const fib = (n) => {
+//   if (n <= 1) {
+//     return n
+//   }
+//   const prev = 0
+//   const cur = 1
+//   for (let i = 2; i <= n; i++) {
+//     const nex = prev + cur
+//     const previous = cur
+//     const current = nex
+//   }
+//   return current
+// }
+// console.log(fib(10))
+
+// #5
+// Write a function range that takes 2 arguments
+// start and end and return an arrey containing
+// all the numbers from the start up to end
+// range (1, 5) => [1, 2, 3, 4, 5]
+
+const strin = []
+function range (a, b) {
+  for (let i = a; i <= b; i++) {
+    strin.push(i)
+  }
+  return strin
+}
+console.log(range(1, 9))
+
+// #6
```

Comment:
Some of your solutions are commented out, I don't know why but I'll assume its rough work or something